Update Admin UI SDK documentation for 1.4.0 release (#160)
AdobeDocs/commerce-extensibility: src/pages/admin-ui-sdk/extension-points/index.md

| Header | Description |
| --- | --- |
| `X-Request-ID` | A unique UUID v4 generated when the request is sent as a unique identifier. |
| `Content-Type` | The request content type is `application/json`. |
| `Accept` | The request accepts a response type is `application/json`. |
| `Authorization` | The request is authenticated using an IMS token generated to the organization attached to the Commerce instance. |
| `x-gw-ims-org-id` | The organization id. |

### Failure on application side

Commerce expects an answer from the application with the error status and message.
The error will be logged in Commerce and an error banner notification will be displayed to user.

### Failure due to connection interruption

The timeout will be applied in case a response is not sent to Commerce. By default it's 10 seconds unless it's customized in the extension point.
When the timeout is reached:

- A 408 timeout status with error message will be logged.
- An error banner notification will be displayed to user.
- An event will be dispatched: `admin_ui_sdk_mass_action_request_failed`. Application can subscribe to this event to take action such as rollback what is already updated in Commerce.

> - Application can access details of the failed request via REST API `V1/adminuisdk/massaction/<requestId>`. A token to Commerce with access to Admin UI SDK rights should be generated. If request is not found, it either do not exist or it succeeded in Commerce.

Reviewer: Make this a separate paragraph, not a bulleted item. 

```suggestion
The application can access details of the failed request using the GET `V1/adminuisdk/massaction/<requestId>` REST API. The token used to authenticate the request must have access to the Admin UI SDK. The call returns an error message if the request ID was not found or if it associated with a successful action.
```

### Recommendations

> - Bulk update in Commerce to avoid inconsistency issues is case of failures.

Reviewer: ```suggestion
- Use bulk update in Commerce to avoid inconsistency issues in case of failures.
```

> - Event and REST API answer contain the list of selected ids for the request. It is the application responsability to monitor what was updated or failed in Commerce.

Reviewer: ```suggestion
- Event and REST API responses contain the list of selected IDs for a request. It is the application's responsibility to monitor updates or failures in Commerce.
```
